.github/CODEOWNERS

```diff
@@ -18,31 +18,33 @@ pkg/postgres/**/*                   @stackrox/core-workflows
 tests/upgrade/*                     @stackrox/core-workflows
 pkg/search/**/*                     @stackrox/core-workflows
 
-# Merlin's main responsibilities include roxctl, authN (authproviders), authZ (SAC), declarative config,
-# performance testing, administration events, cloud sources, and image signatures.
-*/administration/**/*       @stackrox/merlin
-*/auth/**/*                 @stackrox/merlin
-*/cloudsources/**/*         @stackrox/merlin
-*/declarativeconfig/**/*    @stackrox/merlin
-*/discoveredclusters/**/    @stackrox/merlin
-*/signatureintegration/**/* @stackrox/merlin
-pkg/features/**/*           @stackrox/merlin
-pkg/sac/**/*                @stackrox/merlin
-pkg/signatures/**/*         @stackrox/merlin
-roxctl/**/*                 @stackrox/merlin
-tests/performance           @stackrox/merlin
+# sensor-ecosystem main responsibilities include:
+# administration events, authN (authproviders), authZ (SAC)
+# cloud sources, declarative config, image signatures, images, netpol,
+# observability, performance testing, roxctl, sensor and telemetry.
+*/administration/**/*       @stackrox/sensor-ecosystem
+*/auth/**/*                 @stackrox/sensor-ecosystem
+*/cloudsources/**/*         @stackrox/sensor-ecosystem
+*/declarativeconfig/**/*    @stackrox/sensor-ecosystem
+*/discoveredclusters/**/    @stackrox/sensor-ecosystem
+*/signatureintegration/**/* @stackrox/sensor-ecosystem
+pkg/features/**/*           @stackrox/sensor-ecosystem
+pkg/sac/**/*                @stackrox/sensor-ecosystem
+pkg/signatures/**/*         @stackrox/sensor-ecosystem
+roxctl/**/*                 @stackrox/sensor-ecosystem
+tests/performance           @stackrox/sensor-ecosystem
 
-qa-tests-backend/**/*       @janisz
+# Listing all users as "Outside collaborators cannot be added to a team"
+roxctl/netpol/**/*                             @zivnevo @adisos @shireenf-ibm @stackrox/sensor-ecosystem
+tests/roxctl/bats-tests/local/roxctl-netpol-*  @zivnevo @adisos @shireenf-ibm @stackrox/sensor-ecosystem
+tests/roxctl/bats-tests/test-data/np-guard/    @zivnevo @adisos @shireenf-ibm @stackrox/sensor-ecosystem
 
-/ui/**/* @stackrox/ui
+pkg/images/defaults/**/* @stackrox/sensor-ecosystem
+sensor/**/*              @stackrox/sensor-ecosystem
```

Comment:
Move these before outside collaborators?

Reply:
Good point, I sorted our lines, removed duplicates and extracted external contributors